Feature: Additional options (#9)
vtex-apps/trustpilot: docs/README.md

5. When editing the block, add the widget's `template id` and `business unit id` value provided by the [Trustpilot's widget](https://widgets-trustboxinstructions.b2b.trustpilot.com/#/trustbox/eyJjb2RlSGVhZCI6IjwhLS0gVHJ1c3RCb3ggc2NyaXB0IC0tPlxuPHNjcmlwdCB0eXBlPVwidGV4dC9qYXZhc2NyaXB0XCIgc3JjPVwiLy93aWRnZXQudHJ1c3RwaWxvdC5jb20vYm9vdHN0cmFwL3Y1L3RwLndpZGdldC5ib290c3RyYXAubWluLmpzXCIgYXN5bmM%252BPC9zY3JpcHQ%252BXG48IS0tIEVuZCBUcnVzdEJveCBzY3JpcHQgLS0%252BIiwiY29kZUJvZHkiOiI8IS0tIFRydXN0Qm94IHdpZGdldCAtIE1pY3JvIFJldmlldyBDb3VudCAtLT5cbjxkaXYgY2xhc3M9XCJ0cnVzdHBpbG90LXdpZGdldFwiIGRhdGEtbG9jYWxlPVwiaXQtSVRcIiBkYXRhLXRlbXBsYXRlLWlkPVwiNTQxOWI2YThiMGQwNGEwNzY0NDZhOWFkXCIgZGF0YS1idXNpbmVzc3VuaXQtaWQ9XCI1ZTc0ZTJkYjYwMGQxYTAwMDFiZTE1ODlcIiBkYXRhLXN0eWxlLWhlaWdodD1cIjI0cHhcIiBkYXRhLXN0eWxlLXdpZHRoPVwiMTAwJVwiIGRhdGEtdGhlbWU9XCJsaWdodFwiPlxuICA8YSBocmVmPVwiaHR0cHM6Ly9pdC50cnVzdHBpbG90LmNvbS9yZXZpZXcvbWlyaWFkZS5jb21cIiB0YXJnZXQ9XCJfYmxhbmtcIiByZWw9XCJub29wZW5lclwiPlRydXN0cGlsb3Q8L2E%252BXG48L2Rpdj5cbjwhLS0gRW5kIFRydXN0Qm94IHdpZGdldCAtLT4iLCJ0cnVzdEJveE5hbWUiOiJNaWNybyBSZXZpZXcgQ291bnQiLCJsb2NhbGUiOiJpdC1JVCJ9);

> 6. Save your changes.

Reviewer: ```suggestion
6. Save your changes;
```

Author: fixed!

:information_source: _The integration key must be provided by Trustpilot._

The `trustpilot` app also enables Truspilot's widgets to be added to your store theme by inserting the `shop-review-badge` interface in your layout and then using the `trustpilot-widget` block in a template of your choosing.

| Interface           | Block               | Description                             |
| ------------------- | ------------------- | --------------------------------------- |
| `shop-review-badge` | `trustpilot-widget` | Adds a Trustpilot widget in your store. |

## Block without badge

To use widget without `shop-review-badge` to landing pages, for example, you can use `trustpilot-widget`. Therefore, you need to add the following in the manifest.json file.

```diff
 "peerDependencies": {
+   "vtex.trustpilot": "1.x"
 }
```
## `trustpilot-widget` as props

```json

> {
>    "trustpilot-widget#home":{
>     "title":"Widget TrustPilot",
>     "props":{
>       "width":"100%",
>       "height":"240px",
>       "theme":"light",
>       "stars":"4,5",
>       "businessUnitId":"597a23fwffdsfe05a793fe",
>       "templateId": "5419b6a8b0d04a076446a9ad",
>     }
>   }
> }

Reviewer: ```suggestion
{
   "trustpilot-widget#home":{
      "title":"Widget TrustPilot",
      "props":{
         "width":"100%",
         "height":"240px",
         "theme":"light",
         "stars":"4,5",
         "businessUnitId":"597a23fwffdsfe05a793fe",
         "templateId":"5419b6a8b0d04a076446a9ad"
      }
   }
}
```

Author: fixed!

6. Save your changes;
7. Access your store's website using the Developer workspace to check out the new component. If no more changes are desired, repeat the steps above in a [Production workspace](https://vtex.io/docs/recipes/development/creating-a-production-workspace/) and then [promote it to Master](https://vtex.io/docs/recipes/development/promoting-a-workspace-to-master).

> ### Optional

Reviewer 2: ```suggestion
### Optional

```

Author: fixed

> In case that recibe error on toolbet `I couldn't find an interface "shop-review-badge.trustpilot-widget"` after finish all steps to install, you can add the Trustpilot app to your theme's manifest.json file inside peerDependencies as shown below:

Reviewer 2: I couldn't quite understand what you meant here, could you please try to rewrite this section?

Author: I don't know what I meant XD, I didn't understand that that was`shop-review-badge`, i remove this and change for that

```markdown
## Block without badge

To use widget without `shop-review-badge` to landings for example you can use `trustpilot-widget`, you need add to your manifest this....
```

`````diff
 "peerDependencies": {
+   "vtex.trustpilot": "1.x"
 }

> `````

Reviewer 2: There shouldn't be that many `s, 3 are already enough :)

Author: fixed

> 6. Save your changes;

Reviewer 3: A good practice when writing steps is to finish each one with a period or a colon as we can check [here](https://www.notion.so/vtexhandbook/Editorial-guidelines-33cb21cab65844d5b35a7a2e1bf87056#b1748ecd144d42619754c2f7492f3261), for example: 
```suggestion
6. Save your changes.
```

Author: fixed

7. Access your store's website using the Developer workspace to check out the new component. If no more changes are desired, repeat the steps above in a [Production workspace](https://vtex.io/docs/recipes/development/creating-a-production-workspace/) and then [promote it to Master](https://vtex.io/docs/recipes/development/promoting-a-workspace-to-master).

> :information*source: \_The integration key must be provided by Trustpilot*.

Reviewer 3: ```suggestion
:information_source: _The integration key must be provided by Trustpilot._
```

Author: fixed

| ------------------- | ------------------- | --------------------------------------- |
| `shop-review-badge` | `trustpilot-widget` | Adds a Trustpilot widget in your store. |

## Block without badge

> To use widget without `shop-review-badge` to landings for example you can use `trustpilot-widget`, you need add to your manifest this 

Reviewer 3: ```suggestion
To use widget without `shop-review-badge` to landing pages, for example, you can use `trustpilot-widget`. Therefore, you need to add the following in the manifest.json file.
```
Hugo, just to clarify, what do you mean by "landings"? Is there any word could we substitute for? Thank you!

Reviewer 4: I think he meant landing pages

Author: fixed

```diff
 "peerDependencies": {
+   "vtex.trustpilot": "1.x"
 }
```
## `trustpilot-widget` as props

```json
{
   "trustpilot-widget#home":{
    "title":"Widget TrustPilot",
    "props":{
      "width":"100%",
      "height":"240px",
      "theme":"light",
      "stars":"4,5",
      "businessUnitId":"597a23fwffdsfe05a793fe",
      "templateId": "5419b6a8b0d04a076446a9ad",
    }
  }
}
```

| Prop name |   Type   |                         Description                         | Default value |
| :-------: | :------: | :---------------------------------------------------------: | :-----------: |
|  `width`  | `string` | Change width of template |  `100%`   |
|  `height`  | `string` | Change height of template |  `240px`   |
|  `theme`  | `string` | Change theme of template `light` or `dark` |  `light`   |
|  `stars`  | `string` | Change stars stock of template |  `4,5`   |

> |  `businessUnitId`  | `string` | Set your business code  |  null  |

Reviewer 2: Could you update the default values here to `undefined`?

Author: fixed

Author: fixed

| ------------------- | ------------------- | --------------------------------------- |
| `shop-review-badge` | `trustpilot-widget` | Adds a Trustpilot widget in your store. |

## Custom Blocks props of `trustpilot-widget`

> `````json

Reviewer 2: Same think here, just ```json should be enough

Author: fixed

{
   "trustpilot-widget#home":{
    "title":"Widget TrustPilot",
    "props":{
      "width":"100%",
      "height":"240px",
      "theme":"light",
      "stars":"4,5",
      "businessUnitId":"597a23fwffdsfe05a793fe"
    }
  }
}
`````

> | Prop name |   Type   |                         Description                         | Default value |

Reviewer 2: ```suggestion

| Prop name |   Type   |                         Description                         | Default value |
```

Author: fixed
